paddle/fluid/operators/concat_op.cc

```diff
@@ -165,7 +165,7 @@ class ConcatCompositeGradOpMaker : public prim::CompositeGradOpMakerBase {
     paddle::optional<paddle::Tensor> tensor_axis =
         this->GetOptionalSingleForwardInput("AxisTensor");
     paddle::Tensor out_grad = this->GetSingleOutputGrad("Out");
-    std::vector<paddle::Tensor> input_grad = this->GetMultiForwardInput("X");
+    std::vector<paddle::Tensor> input_grad = this->GetMultiInputGrad("X");
```

Comment:
change to `std::vector<paddle::Tensor> input_grad = this->GetMultiInputGrad("X", false);`

Reply:
done